devtools/dev_container/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:20.04
+
+# Adding rust binaries to PATH.
+ENV PATH="$PATH:/root/.cargo/bin"
+ENV CC=clang
+ENV AR=llvm-ar
+
+# Install all required packages in one go to optimize the image
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run
+# DEBIAN_FRONTEND is set for tzdata.
+RUN apt-get update && \
+    DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -y \
+    build-essential ca-certificates curl gcc git libssl-dev pkg-config ssh \
+    clang llvm nasm \
+    # cleanup
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install rustup and a fixed version of Rust.
```

Comment:
I don't mind, but shouldn't you do a run with multiple steps just to be consistent with the comments on lines 8-10? 😅

Reply:
The dev container is unstable yet, so keeping them in separate lines helps to use the build cache.
We may combine them into one line once it gets stable:)

Reply:
OK. Makes sense.